docker-compose.yml

```diff
@@ -4,24 +4,14 @@ services:
 
   backend:
     build:
-      context: .
-      dockerfile: ./backend/Dockerfile
+      context: ./backend
     container_name: django_backend
-    volumes:
-      - ./backend/:/app
     ports:
       - "8000:8000"
-    command: python manage.py runserver 0.0.0.0:8000
 
   frontend:
     build:
-      context: .
-      dockerfile: ./frontend/Dockerfile
-    image: nuxt_dev
```

Comment:
I believe `image` just gives a custom name to the built image. It's not necessary really, so I wasn't sure about keeping it, since the same isn't done for the backend image.

Reply:
Let's keep it simple I guess :)